humblebundle.py

```diff
@@ -728,10 +736,15 @@ def main(argv=None):
         hb.update()
 
     if args.list is not None:
+        if args.platform is not None:
+            keys = filterGames(hb.games,args.platform)
```

Comment:
I believe the whole `filterGames()` function can be replaced by a one-line, such as:
`keys = [keys for keys in games if any(_ for _ in games[keys]['downloads'] if _['platform'] == args.platform)]`